libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -102,6 +105,20 @@ const AP_Param::GroupInfo SRV_Channels::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO_FRAME("_AUTO_TRIM",  17, SRV_Channels, auto_trim, 0, AP_PARAM_FRAME_PLANE),
     
+    // @Param: _CANSRV_BM
+    // @DisplayName: RC Out channels to be transmitted as servo over CAN bus
+    // @Description: Bitmask with one set for channel to be transmitted as a servo command over CAN bus
+    // @Bitmask: 0: Servo 1, 1: Servo 2, 2: Servo 3, 3: Servo 4, 4: Servo 5, 5: Servo 6, 6: Servo 7, 7: Servo 8, 8: Servo 9, 9: Servo 10, 10: Servo 11, 11: Servo 12, 12: Servo 13, 13: Servo 14, 14: Servo 15
```

Comment:
UAVCAN have different commands for Servo and ESC. That is why it clearly stated this is not for channel in overall but specifically for Servo.